apps/nextjs/src/app/page.tsx

```
@@ -12,24 +13,44 @@ export default function Home() {
+        <h2 className={`${title({ color: "violet", size: "xs" })} pt-4`}>
+          Coming Soon
+        </h2>
+      </div>
+    </div>
+  );
+}
+
+function FrontPageTextSkeleton() {
+  return (
+    <div className="flex w-full max-w-fit animate-pulse flex-col items-center justify-center bg-transparent text-center">
+      <div className="h-8 w-48 rounded bg-gray-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)